### DISCUSSION TITLE
git clone issue

### DISCUSSION THREAD

**Discussion by FatemehAskari** [community/community#53359] [Repositories] [answered]
### Select Topic Area

Question

### Body

I am unable to clone a repository from GitHub.
C:\Users\user>git clone https://github.com/boppreh/aes.git
Cloning into 'aes'...
fatal: unable to access 'https://github.com.cnpmjs.org/boppreh/aes.git/': Could not resolve host: github.com.cnpmjs.org

**Comment by thomasbnt** [comment, marked as answer]
Hello @FatemehAskari !

You can clone the repository with 
```bash
git clone https://github.com/boppreh/aes.git
```
Or by replacing `https` by `git` : 

```bash
git clone git://github.com/boppreh/aes.git
```

Check that you are not blocking the request with third party software.
